examples/Rakefile

```diff
@@ -1,41 +1,111 @@
 require 'fastly_nsq'
 require 'fastly_nsq/rake_task'
```

Comment:
Yeah the general `require 'fastly_nsq'` I believe is not required - we only want the rake task here, and it's only dependency is on `MessageProcessor`. I've removed in 1d23dd7